website/components/HowToGuideIndex.tsx

```diff
@@ -0,0 +1,45 @@
+import { getPagesUnderRoute } from "nextra/context";
+import Link from "next/link";
+
+interface Page {
+  name: string;
+  route: string;
+  children?: Page[];
+  meta?: Record<string, any>;
+  frontMatter?: any;
+}
+
+function sortByTitle(a: Page, b: Page) {
+  return (
+     b.frontMatter?.title?.toLowerCase() < a.frontMatter?.title?.toLowerCase() ? 1 : -1
+  );
+}
+
+export function HowToGuideIndex({ more = "Read the guide" }) {
```

Comment:
Is this one necessary? i.e can't we use the same thing built-in theme as for the blog?

Reply:
We probably can if we provide the blog index with some more options, but the ordering is different for how-to guides (they currently are ordered by title, not by date)

Reply:
Can we remove the dates from the how to guides? I think that will ensure they get sorted by titles

Reply:
Removed 👍